docker-compose.yaml

```diff
@@ -0,0 +1,20 @@
+# version: '3.8'
+services:
+  axolotl:
+    build:
+      context: .
+      dockerfile: ./docker/Dockerfile
+    volumes:
+      - .:/workspace/axolotl
+      - ~/.cache/huggingface/:/root/.cache/huggingface/
+    # set environment variables
+    environment:
+      - WANDB_API_KEY=${WANDB_API_KEY}
+    deploy:
+      resources:
+        reservations:
+          devices:
+            - driver: nvidia
+              # count: 1
+              capabilities: [gpu]
+    command: tail -f /dev/null
```

Comment:
is this the correct command for local dev?

Reply:
`command: tail -f /dev/null` keeps the container running otherwise it will shutdown immediately

Reply:
👍 sgtm, I may change it to use sleep infinity down the line, https://stackoverflow.com/a/35770783